handlers/helpers.js

```diff
@@ -1,83 +1,110 @@
 
-module.exports = {    
-    isEmpty: function(obj) {
-        return Object.keys(obj).length === 0;
-    },
+module.exports = {
+  isEmpty: function(obj) {
+    return Object.keys(obj).length === 0;
+  },
 
-    inputError: function(message, data) {
-        return {
-            statusCode: 406,
-            headers: {
-              'Access-Control-Allow-Origin': '*',
-              'Access-Control-Allow-Credentials': true,
-            },
-            body: JSON.stringify({
-               message: message,
-               data: data
-            }, null, 2),
-          };
-    },
+  inputError: function(message, data) {
+    const response = {
+      statusCode: 406,
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Credentials": true
+      },
+      body: JSON.stringify(
+        {
+          message: message,
+          data: data
+        },
+        null,
+        2
+      )
+    };
+    console.log("INPUT ERROR", response);
+    return response;
+  },
 
-    /**
-     * 
-     * @param {*} id - String or Integer item ID
-     * @param {Object} obj - object containing key value paris
-     * @param {String} table - name of table, ie 'biztechUsers'
-     */
-    updateDB: async function(id, obj, table) {        
-        var updateExpression = 'set ';
-        var expressionAttributeValues = {};
-
-        // TODO: Add a filter for valid object keys
-        // loop through keys and create updateExpression string and
-        // expressionAttributeValues object
-        for (var key in obj){
-            if(obj.hasOwnProperty(key)) {
-                if (key != 'id'){
-                    updateExpression += key + '\= :' + key + ',';
-                    expressionAttributeValues[':' + key] = obj[key];
-                }
-            }
+  /**
+   *
+   * @param {Array} batch - List of batches in form of [{Key: value}]
+   * @param {String} tableName - Name of table to call batchGet
+   */
+  batchGet: function(batch, tableName) {
+    const batchRequestParams = {
+      RequestItems: {
+        [tableName]: {
+          Keys: batch
         }
+      }
+    };
 
-        const timestamp = new Date().getTime();
-        updateExpression += "updatedAt = :updatedAt";
-        expressionAttributeValues[':updatedAt'] = timestamp;
+    console.log('BatchRequestParams', batchRequestParams)
 
-        var params = {
-            Key: { id },
-            TableName: table + process.env.ENVIRONMENT,
-            ExpressionAttributeValues: expressionAttributeValues,
-            UpdateExpression: updateExpression,
-            ReturnValues:"UPDATED_NEW"
-        };
+    return docClient.batchGet(batchRequestParams).promise()
+  },
+
+  /**
+   *
+   * @param {*} id - String or Integer item ID
+   * @param {Object} obj - object containing key value paris
+   * @param {String} table - name of table, ie 'biztechUsers'
+   */
+  updateDB: async function(id, obj, table) {
```

Comment:
Did you change updateDB or is this just git being funny?

Reply:
I just ran prettify on this file